Corrections/btagsf.cc

```diff
@@ -1,42 +1,29 @@
-#include "BTagCalibrationStandalone.h"
-#include "BTagCalibrationStandalone.cpp"
+// with CMSSW:
+#include "CondFormats/BTauObjects/interface/BTagCalibration.h"
+#include "CondTools/BTau/interface/BTagCalibrationReader.h"
 
 #include <cmath>
 using namespace ROOT::VecOps;
 
 namespace analyzer {
-    std::vector<float> btagSF(string year, TLorentzVector* b_jet0,TLorentzVector* b_jet1) {
-    	std::vector<float> v;
-	    if (year == '16'){
-            BTagCalibration calib('DeepCSV', 'SFs/DeepCSV_2016LegacySF_V1.csv');
-        }elif (year == '17'){
-            BTagCalibration calib('DeepCSV', 'SFs/DeepCSV_94XSF_V4_B_F.csv');
-        }elif (year == '18'){
-            BTagCalibration calib('DeepCSV', 'SFs/DeepCSV_102XSF_V1.csv');
-        }
+  std::vector<float> BtagSF(BTagCalibrationReader reader,TLorentzVector* b_jet0,TLorentzVector* b_jet1) {
```

Comment:
Your taking two jets as arguments but only doing the calculation on b_jet1. In fact, we only want to do one at a time so I think b_jet1 should be dropped, b_jet0 should be changed to just b_jet, and the corresponding variable names below should be changed.